src/Didomi.ts

```diff
@@ -285,6 +285,12 @@ export const Didomi = {
       vendorId
     ),
 
+  /**
+   *  Get the user consent status.
+   *  @returns: status that represents user consent.
+   */
+  getUserStatus: (): Promise<any> => RNDidomi.getUserStatus(),
```

Comment:
We should also deprecate the other methods, with a message indicating the equivalent using getUserStatus

Reply:
getUserConsentStatusForPurpose, etc.